frontend/src/utils/formatting.js

```diff
@@ -13,19 +13,37 @@
  * See more on the formatting options, here:
  * https://github.com/nmn/react-timeago#formatter-optional
  */
-const yearAndDayFormatter = (value, unit, suffix, epochMiliseconds) => {
+const yearAndDayFormatter = (value, unit, suffix, elapsedMilliseconds) => {
+  // Only apply custom logic for the year case
   if (unit === "year") {
     // The days calculation comes directly from react-timeago:
-    const days = Math.round(Math.abs(epochMiliseconds - Date.now()) / (1000 * 60 * 60 * 24));
+    // https://github.com/nmn/react-timeago/blob/master/src/formatters/buildFormatter.js#L84-L86
+
+    // Number of ms in a second: 1000
+    // Number of seconds in a minute: 60
+    // Number of minutes in an hour: 60
+    // Number of hours in a day: 24
+    // Should be 86,400,000
+    const msInDays = 1000 * 60 * 60 * 24;
+    const elapsedDaysSinceEpoch = Math.round(elapsedMilliseconds / msInDays);
```

Comment:
`elapsedMilliseconds` isn't the diff between now and the date we're formatting. It's the total unix time elapsed until the date that we're formatting. I think you want the time elapsed between the date you're trying to format and now here.